src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -993,7 +993,7 @@
     short-summary: "Describe how VMs are added to or removed from nodepools."
   - name: --node-taints
     type: string
-    short-summary: The node taints for the node pool.
+    short-summary: The node taints for the node pool. Pass the empty string `""` to remove all taints.
```

Comment:
Though it's technically correct, (help message for creating resource) when adding a new nodepool, if user does not specify this option, the default node taint is empty.

Reply:
I agree with @FumingZhang. For adding commands, `--node-taints ""` is redundant.
```suggestion
    short-summary: The node taints for the node pool.
```

Reply:
Definitely makes sense 👍